etc/publish.sh

```diff
@@ -4,6 +4,18 @@ set -e
 
 BP_NAME=${1:-"heroku/nodejs"}
 
+# if buildpack-registry CLI plugin is not installed, show a help message and exit
+if heroku plugins | grep -qv "buildpack-registry"; then
```

Comment:
Don't we want this to show if it doesn't exist?

Reply:
`-v` inverts this logic, so it will be true when `buildpack-registry` is not present. I'll switch it to `--invert-match` which is clearer